pkgs/by-name/po/polybar-pulseaudio-control/package.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, bash
+, coreutils
+, fetchFromGitHub
+, gawk
+, pulseaudio
+, stdenvNoCC
+, writeShellApplication
+}:
+
+let
+  polybar_pactl = stdenvNoCC.mkDerivation (finalAttrs: {
+    pname = "polybar-pulseaudio-control";
+    version = "3.1.1";
+
+    src = fetchFromGitHub {
+      owner = "marioortizmanero";
+      repo = "polybar-pulseaudio-control";
+      rev = "v${finalAttrs.version}";
+      hash = "sha256-egCBCnhnmHHKFeDkpaF9Upv/oZ0K3XGyutnp4slq9Vc=";
+    };
+
+    dontConfigure = true;
+    dontBuild = true;
+
+    installPhase = ''
+      runHook preInstall
+
+      install -Dm755 pulseaudio-control.bash -t $out/bin
+
+      runHook postInstall
+    '';
+
+  });
+in
+writeShellApplication {
+  name = "polybar-pulseaudio-control";
+  runtimeInputs = [ polybar_pactl bash coreutils gawk pulseaudio ];
+  text = ''
+    pulseaudio-control.bash "$@"
+  '';
+  meta = with lib; {
+    mainProgram = "pulseaudio-control";
+    description = "Polybar module to control PulseAudio devices";
+    homepage = "https://github.com/marioortizmanero/polybar-pulseaudio-control";
+    platforms = platforms.linux;
+    license = licenses.mit;
+    maintainers = with maintainers; [ benlemasurier ];
+  };
+}
```

Comment:
Does this derivation exist for the sole reason of adding these runtime inputs, and renaming the command? In that case, I think `makeWrapper` would be a better solution.